sympy/stats/joint_rv_types.py

```diff
@@ -253,3 +253,71 @@ def NormalGamma(syms, mu, lamda, alpha, beta):
     A random symbol
     """
     return multivariate_rv(NormalGammaDistribution, syms, mu, lamda, alpha, beta)
+
+
+#-------------------------------------------------------------------------------
+# Multivariate Beta distribution ---------------------------------------------------------
+
+class MultivariateBetaDistribution(JointDistribution):
+
+    _argnames = ['alpha']
+    is_Continuous = True
+
+    def check(self, alpha):
+        _value_check(len(alpha) >= 2, "At least two categories should be passed.")
+        _value_check(all([a_k > 0 for a_k in alpha]), "Each concentration parameter"
+                                                        " should be positive")
+
+    @property
+    def set(self):
+        k = len(self.alpha)
+        return Interval(0, 1)**k
+
+    def pdf(self, *syms):
+        alpha = self.alpha
+        B = Mul(*[gamma(a_k) for a_k in alpha])/gamma(Add(*[a_k for a_k in alpha]))
+        return Mul(*[sym**(a_k) for a_k, sym in zip(alpha, syms)])/B
+
+def MultivariateBeta(syms, alpha):
+    """
+    Creates a continuous random variable with Dirichlet/Multivariate Beta
+    Distribution.
+
+    The density of the dirichlet distribution can be found at [1].
+
+    Parameters
+    ==========
+
+    alpha: List of positive real numbers signifying concentration numbers.
+
+    Returns
+    =======
+
+    A RandomSymbol.
+
+    Examples
+    ========
+
+    >>> from sympy.stats import density
+    >>> from sympy.stats.joint_rv import marginal_distribution
+    >>> from sympy.stats.joint_rv_types import MultivariateBeta
+    >>> from sympy import Symbol
+    >>> a1 = Symbol('a1', positive=True, real=True)
+    >>> a2 = Symbol('a2', positive=True, real=True)
```

Comment:
`real=True` can be dropped while defining `a1` and `a2` as they are already implied by `positive=True`.